bundles/SyliusReviewBundle/installation.rst

```diff
@@ -0,0 +1,82 @@
+Installation
+============
+
+We assume you're familiar with `Composer <http://packagist.org>`_, a dependency manager for PHP.
+Use the following command to add the bundle to your `composer.json` and download the package.
+
+If you have `Composer installed globally <http://getcomposer.org/doc/00-intro.md#globally>`_.
+
+.. code-block:: bash
+
+    $ composer require "sylius/review-bundle"
+
+Otherwise you have to download .phar file.
+
+.. code-block:: bash
+
+    $ curl -sS https://getcomposer.org/installer | php
+    $ php composer.phar require "sylius/review-bundle"
+
+Adding required bundles to the kernel
+-------------------------------------
+
+You need to enable the bundle inside the kernel.
+
+If you're not using any other Sylius bundles, you will also need to add `SyliusResourceBundle` and its dependencies to kernel.
```

Comment:
Im a little bit picky, but I think *you're not* is not a good phrase. I would use *you are not* or *you aren't*